packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -251,9 +265,10 @@ const FooterTagList = ({
 
   const contentTypeInfo = forumSelect(contentTypes);
 
-  const PostTypeTag = useCallback(({tooltipBody, label}: {
+  const PostTypeTag = useCallback(({tooltipBody, label, neverCoreStyling}: {
```

Comment:
I really think this should be pulled out

Reply:
I'm slightly confused about this. Looks like @oetherington wrote the original code, curious if there are reasons for it being here.

Reply:
I don't remember any particular reason. I think it just got refactored from somewhere else and was small enough to feel like this was fine. As it gets bigger and more complicated it definitely feels reasonable to make it a separate component.